Blockzilla/AppDelegate.swift

```diff
@@ -207,7 +207,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
 
     fileprivate func displaySplashAnimation() {
         let splashView = self.splashView
-        splashView.backgroundColor = UIConstants.colors.background
+        splashView.backgroundColor = UIConstants.colors.settingsBackgroundColor
```

Comment:
Nit: since we're now using the constant settingsBackgroundColor outside of just settings and it is our default background color now, I think we should rename to just backgroundColor for clarity.

Reply:
Yep, was considering this. I'll change it :)